### PR TITLE
Into to server inlets branch

### PR DESCRIPTION
**Pull request by anfee1**
Adds new servlet and uses json.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/UsingJson.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/Team")
+public class UsingJson extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("application/json;");
+    String team = "{\"members\": [{\"name\": \"Chris\", \"status\" : \"FTE\"},{\"name\": \"Ben\", \"status\": \"FTE\"},{\"name\": \"Cindy\", \"status\" : \"Intern\"},{\"name\": \"Charles\", \"status\" : \"Intern\"}]}";
+    response.getWriter().println(team);
+    }
```
Comment (KoopaKing) [inline portfolio/src/main/java/com/google/sps/servlets/UsingJson.java:31]: I think you need to run the formatter on this.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -53,3 +53,16 @@ async function setHelloContainerWithServlet() {
...
+
+    var teamParagraph = 'Here are the members of my team and their position:\n';
+
+    for (i = 0; i < theTeam.members.length; i++){
+        teamParagraph =  teamParagraph + theTeam.members[i].name + '\t'+ theTeam.members[i].status + '\n'
+    }
+
+    document.getElementById('team-container').innerText = teamParagraph;
+ });
+}
```
Comment (KoopaKing) [inline portfolio/src/main/webapp/script.js:68]: Add a newline EOF

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -120,13 +123,25 @@ <h1>Anthony's Google  Portfolio</h1>
...
+        <section id="smallContent3">
+            <h1>My Team</h1>
+            <p>Click here to see my team members:</p>
+            <button onclick="setMyTeamConatainerWithServerlet()">Get Team</button>
+            <div id="team-container"></div>
+        </section>
+        <section id="smallContent3">
+            <h1>Hello Message</h1>
+            <p>Click here to get a hello:</p>
+            <p>This is a filler box idk what I'll add to this</p>
```
Comment (KoopaKing) [inline portfolio/src/main/webapp/index.html:143]: nit: this seems like two separate sentences.

File: portfolio/src/main/java/com/google/sps/servlets/UsingJson.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/Team")
+public class UsingJson extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("application/json;");
+    String team = "{\"members\": [{\"name\": \"Chris\", \"status\" : \"FTE\"},{\"name\": \"Ben\", \"status\": \"FTE\"},{\"name\": \"Cindy\", \"status\" : \"Intern\"},{\"name\": \"Charles\", \"status\" : \"Intern\"}]}";
```
Comment (KoopaKing) [inline portfolio/src/main/java/com/google/sps/servlets/UsingJson.java:29]: nit: Inlining the data payload is gross. You could throw this into a file that you read from or something, but that's not necessary for this pull.

**Comment by anfee1** [comment]
[https://afeenster-step-2020.ue.r.appspot.com/](url)

link to my website... This should work.

**Pull request merged**